app/scripts/modules/netflix/pagerDuty/pagerDutyTag.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import {Observable} from 'rxjs';
+import {DebugElement, SimpleChange} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {IPagerDutyService, PagerDutyReader} from './pagerDuty.read.service';
+import {PagerDutyTagComponent} from './pagerDutyTag.component';
+
+describe('PagerDutyTagComponent', () => {
+
+  let component: PagerDutyTagComponent;
+  let fixture: ComponentFixture<PagerDutyTagComponent>;
+  let debugElement: DebugElement;
+  let htmlElement: HTMLElement;
+
+  const services: IPagerDutyService[] = [
+    {name: 'name1', integration_key: 'integrationKey1'},
+    {name: 'name2', integration_key: 'integrationKey2'},
+    {name: 'name3', integration_key: 'integrationKey3'}
+  ];
+  const pagerDutyReaderStub = {
+    listServices: () => {
+      return Observable.of(services);
+    }
+  };
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [PagerDutyTagComponent]
+    }).overrideComponent(PagerDutyTagComponent, {
+      set: {
+        providers: [
+          {provide: PagerDutyReader, useValue: pagerDutyReaderStub}
+        ]
+      }
+    });
+
+    fixture = TestBed.createComponent(PagerDutyTagComponent);
+    component = fixture.componentInstance;
+    debugElement = fixture.debugElement.query(By.css('span'));
+    htmlElement = debugElement.nativeElement;
+  }));
+
+  it('should return the mocked pager duty services', (done) => {
+    const reader: PagerDutyReader = debugElement.injector.get(PagerDutyReader);
+    reader.listServices().subscribe((values: IPagerDutyService[]) => {
+      expect(values).toBe(services);
+      done();
+    });
+  });
```

Comment:
Is this testing anything valuable? Seems like the other tests validate that the mocked service is being used.